### ISSUE TITLE
cartridge prevents collectstatic command from running when using ManifestStaticFilesStorage

### ISSUE THREAD

**Issue by jlirochon** [stephenmcd/cartridge#317]
**cartridge** makes **collectstatic** command throw an exception when using ```django.contrib.staticfiles.storage.ManifestStaticFilesStorage``` : 

```
ValueError: The file 'cartridge/js/admin/product_variations.js' could not be found with <django.contrib.staticfiles.storage.ManifestStaticFilesStorage object at 0x1026ea208>.
```
----
### 1. Install **cartridge**. Today doing ```pip install cartridge``` is the same as installing the following from **requirements.txt** :

```
#
# This file is autogenerated by pip-compile
# To update, run:
#
#    pip-compile --output-file requirements.txt requirements.in
#
beautifulsoup4==4.5.3     # via mezzanine
bleach==1.5.0             # via mezzanine
cartridge==0.12.0
chardet==2.3.0            # via mezzanine
django-contrib-comments==1.7.3  # via mezzanine
django==1.10.5            # via django-contrib-comments, mezzanine
filebrowser-safe==0.4.6   # via mezzanine
future==0.16.0            # via mezzanine
grappelli-safe==0.4.5     # via mezzanine
html5lib==0.9999999       # via bleach, xhtml2pdf
mezzanine==4.2.2          # via cartridge
oauthlib==2.0.1           # via requests-oauthlib
olefile==0.44             # via pillow
Pillow==4.0.0             # via mezzanine, reportlab, xhtml2pdf
pyPdf2==1.26.0            # via xhtml2pdf
pytz==2016.10             # via tzlocal
reportlab==3.3.0          # via xhtml2pdf
requests-oauthlib==0.7.0  # via mezzanine
requests==2.12.5          # via mezzanine, requests-oauthlib
six==1.10.0               # via bleach, html5lib
tzlocal==1.3              # via mezzanine
webencodings==0.5
xhtml2pdf==0.0.6          # via cartridge

# The following packages are considered to be unsafe in a requirements file:
# pip                       # via reportlab
# setuptools                # via reportlab
```

### 2. Then create a new project : 

```
mezzanine-project -a cartridge cartridge_debug
```

### 3. Then put this line in your **settings.py** :

```
STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'
```

### 4. Then disable **DEBUG** in your **local_settings.py**

```
DEBUG = False
```

### 5. Finally run **collectstatic** :

```
python manage.py collectstatic --noinput
```

**Comment by jlirochon** [comment]
I can reproduce the same behavior with django **1.8.17** and **1.9.12**

**Comment by ryneeverett** [comment]
Resolved by #318.
